diff --git a/tensorflow/lite/experimental/ruy/platform.h b/tensorflow/lite/experimental/ruy/platform.h
index 7a83173a..dd5fe93a 100644
--- a/tensorflow/lite/experimental/ruy/platform.h
+++ b/tensorflow/lite/experimental/ruy/platform.h
@@ -49,16 +49,8 @@ limitations under the License.
 #define RUY_DONOTUSEDIRECTLY_NEON_64 \
   (RUY_DONOTUSEDIRECTLY_NEON && RUY_DONOTUSEDIRECTLY_ARM_64)
 
-// These CPU capabilities will all be true when Skylake is enabled during
-// compilation.
-//
-// TODO(b/138433137) Select AVX-512 at runtime rather than via compile options.
-#if defined(__AVX512F__) && defined(__AVX512DQ__) && defined(__AVX512CD__) && \
-    defined(__AVX512BW__) && defined(__AVX512VL__)
-#define RUY_DONOTUSEDIRECTLY_AVX512 1
-#else
+// Disable AVX512 Optimization
 #define RUY_DONOTUSEDIRECTLY_AVX512 0
-#endif
 
 // Detect APPLE
 #ifdef __APPLE__
